src/content/content.service.ts

```
@@ -262,106 +302,65 @@ export class ContentService {
-				errorMessage: error.message,
-				// Attach the data in the "data" field
-			});
 		}
+		
+		console.log('arrayOfObjects length', arrayOfObjects.length);
+		return arrayOfObjects;
 	}
 
+	private transformProviderItems(provider: any, context: any): any[] {
+		return provider.items.map((item) => ({
+			unique_id: this.generateFixedId(item.id, item.descriptor.name, context.bpp_id),
```

ESLint

[error] Replace `item.id,·item.descriptor.name,·context.bpp_id` with `⏎↹↹↹↹item.id,⏎↹↹↹↹item.descriptor.name,⏎↹↹↹↹context.bpp_id,⏎↹↹↹`

(prettier/prettier)